src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -71,4 +78,27 @@ class Lambda {
     val purged = response.code == 200
     purged
   }
+
+  /**
+   * Send a ping request to Google AMP to refresh the cache.
+   * See https://developers.google.com/amp/cache/update-ping
+   *
+   * @return whether the request was successfully processed by the server
+   */
+  private def sendAmpPingRequest(contentId: String, purgeType: PurgeType): Boolean = {
+    val contentPath = s"/$contentId"
+
+    val url = s"https://amp-theguardian-com.cdn.ampproject.org/c/s/amp.theguardian.com${contentPath}"
```

Comment:
@NataliaLKB sure, I have updated to the lastest one in the docs https://ampbyexample.com/advanced/using_the_google_amp_cache/